app/server/controllers/login.ts

```diff
@@ -298,3 +310,19 @@ export function setupLoginController(deps: Dependencies): Router {
   router.get(Paths.logout, getLogout(deps.idamService));
   return router;
 }
+
+function getReducedEmailforLogs(email) {
+  let shortenedEmail = '';
+  if (email !== null) {
+    const indexOfAt = email.indexOf('@');
+    const indexOfFinalChar = parseInt(indexOfAt) + 3;
+    shortenedEmail = shortenedEmail.concat(
+      email.substring(0, 3),
+      '...',
```

Comment:
is the requirement to use dots? to me dots might seem like the string has been truncated rather than masked
Asterix (*) might be more suitable